src/app/components/footer/footer.component.scss

```diff
@@ -0,0 +1,18 @@
+.footer {
+  /* display */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  /* positioning */
+  position: fixed;
+  bottom: 0;
+
+  /* box model */
+  width: 100%;
+  height: 2rem;
+  background-color: #5460ac;
+
+  /* type */
```

Comment:
I'm not sure if those comments are necessary. If something can't be automated by IDE or tools it is adding a bunch of overhead and is decreasing developer experience because that way developer needs to learn one more convention that exist in the project.
I would  create a ticket to maybe do some research if SCSS code organisation can be automated or ignored this convention and sorting properties.